src/main/java/com/thevoxelbox/voxelsniper/brush/GenerateTreeBrush.java

```diff
@@ -189,18 +144,22 @@ private void leafNodeCreate() {
         }
     }
 
+    private void createLeaf(int x, int y, int z) {
+        if (location.getOffsetBlock(x, y, z).getType() == Material.AIR) {
+            this.undo.put(location.getOffsetClampedBlock(x, y, z));
+            location.getOffsetBlock(x, y, z).setBlockData(leavesMaterial.createBlockData(), false);
+        }
+    }
+
     /**
      * Code Concerning Root Generation.
      *
-     * @param xDirection
-     * @param zDirection
+     * @ param xDirection
+     * @ param zDirection
```

Comment:
```suggestion
     * @param xDirection
     * @param zDirection
```
I don't think there should be spaces here??